packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -32,6 +38,22 @@ export class HttpForwarder
     };
   }
 
+  private updateHostHeaders(baseUrl: string, headers?: IHttpNameValue) {
+    // no headers? do nothing
+    if (!headers) return headers;
+
+    // host header provided? override with actual hostname
+    if (headers.hasOwnProperty('host')) {
```

Comment:
I _think_ that `host` is a mandatory http header or something — basically the client can't route the request at all without it. If this code is executed, the `host` header is definitely there.

Maybe this check can be removed (but maybe Typescript will start to complain?)

Reply:
Just checked (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host).

Host: is required for all HTTP/1.1 reqs. That means that there _are_ some requests in the world that do not set Host header ;)

Additionally I bet all webservers will respond properly when Host: is not set (e.g. apache and nginx will respond with default virtual server).

Reply:
Right, that is a good point. `io-ts` would be really what we need here :trollface: